demo/playground/README.md

```diff
@@ -0,0 +1,124 @@
+# Aca-Py Playground
+
+This directory contains scripts to run several Aca-Py agents in various configurations for demonstration, development and testing scenarios. The agents are using Postgres (15) database storage (`askar`, `DatabasePerWallet`) and are running without security (`--admin-insecure-mode`).
+
+The inspiration for this playground was testing mediation and the differences between single-tenant and multi-tenant modes. These scripts allowed the developer to stand up 3 single-tenant agents and 1 multi-tenant agent and run various scenarios (in other scripts - not included). Running in `--admin-insecure-mode` simplifies creating tenants in multi-tenant mode and eliminates the need for adding headers for calls in single-tenant mode.
```

Comment:
Would it make sense to include one script as an example so it’s easier for others to be built?

Reply:
will do. i'll keep it self contained (ie. no mediator).